downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -38,10 +38,46 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 ## Runtime updates
 
-### New features
-
 ### Improvements
 
+#### Added shift, unshift, pop and remove operation support to tuples
```

Comment:
```suggestion
#### Added support for the `shift`, `unshift`, `pop`, and `remove` functions with tuples

```

Reply:
Thank you for the suggestions.
Fixed with fe2b9b455480bb52a4722bd586efd701df85f10b